routers/products-router.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express')
+const ProductsRouter = express.Router();
+const {Products} = require('../models/products-model')
+
+
+ProductsRouter.route('/')
+.get(async (req,  res) =>{
+  // get all
+  try{
+    
+    const myData = await Products.find({})
+    console.log({myData})
+    res.json({myData : myData })
+  }
+  catch(error){
+    res.json({status : false , message : "failed" , errMessage : error.message})
+  }
+
+})
+.post(async (req  , res) => {
+  try{
+    const data = req.body
+    console.log(data)
+    const NewProduct = new Products(data)
+    const SavedProduct =await  NewProduct.save()
+    res.json({status : true  , SavedProduct})
+  }catch(error){ res.status(500).json({status : false  ,
```

Comment:
That is how you should do in above code.

Reply:
yes did that , thanks for the heads up